jicofo-common/src/main/java/org/jitsi/impl/protocol/xmpp/ChatRoomImpl.kt

```diff
@@ -127,17 +130,13 @@ class ChatRoomImpl(
     /** The number of members that currently have their video sources unmuted. */
     private var numVideoSenders = 0
 
-    override fun getRoomJid() = roomJid
-    override fun getXmppProvider() = xmppProvider
-    override fun isBreakoutRoom() = isBreakoutRoom
-    override fun getMainRoom() = mainRoom
-    override fun getMeetingId(): String? = meetingId
     override fun addListener(listener: ChatRoomListener) = eventEmitter.addHandler(listener)
     override fun removeListener(listener: ChatRoomListener) = eventEmitter.removeHandler(listener)
-    override fun isJoined() = muc.isJoined
-    override fun getMembers(): List<ChatRoomMember> = synchronized(members) { return members.values.toList() }
-    override fun getChatMember(occupantJid: EntityFullJid) = members[occupantJid]
-    override fun getMembersCount() = members.size
+    override val isJoined
+        get() = muc.isJoined
+    override val members: List<ChatRoomMember>
```

Comment:
Removing `membersSize` from the interface means that calls to `members.size` construct the full `members` list before returning its size.  How often do we call that?  Is this a performance consideration?

Reply:
Oh, good point. I think that's why we introduced `getMembersCount()` in the first place 🤦